.github/workflows/prettier.yml

```diff
@@ -0,0 +1,24 @@
+name: Prettier Markdown
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - 'docs/**'
+
+jobs:
+  pretty:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+      - run: npm install -g yarn
+      - run: yarn install
+      - name: Prettify code
+        uses: creyD/prettier_action@v3.1
+        with:
+          prettier_options: --config docs/prettier.config.js --write docs/**/*.md
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_SERVICE_ACCOUNT_TOKEN }}
```

Comment:
Better use the standard workflow token here rather than this, or we may trigger a loop

Reply:
Thanks for the feedback, will action all changes , could you tell me what is the standard workflow token ? I will change the token i have with it as soon as i get it

Reply:
I tried secrets.GITHUB_TOKEN but it failed when checking out the code